dev-libs/mauikit-filebrowsing/mauikit-filebrowsing-9999.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_QTHELP="true"
+KDE_ORG_CATEGORY="maui"
+KFMIN=5.82.0
+QTMIN=5.15.5
+inherit ecm kde.org
+
+DESCRIPTION="MauiKit File Browsing utilities and controls"
+
+LICENSE="GPL-3"
+SLOT="5"
+KEYWORDS=""
+IUSE=""
+
+DEPEND="
+	sys-devel/gettext
+	>=dev-qt/qtsql-${QTMIN}:5
+	>=dev-qt/qtxml-${QTMIN}:5
+	>=dev-qt/qtcore-${QTMIN}:5
+	>=dev-qt/qtnetwork-${QTMIN}:5
+	>=dev-qt/qtconcurrent-${QTMIN}:5
+	>=dev-qt/qtdeclarative-${QTMIN}:5
+
+	>=dev-libs/mauikit-${KFMIN}:5
+	>=kde-frameworks/kio-${KFMIN}:5
+	>=kde-frameworks/ki18n-${KFMIN}:5
+	>=kde-frameworks/kcoreaddons-${KFMIN}:5
+	>=kde-frameworks/kconfig-${KFMIN}:5[qml]
+	>=kde-frameworks/extra-cmake-modules-${KFMIN}:5
```

Comment:
ECM is never an `RDEPEND`, and `ecm.eclass` already `BDEPEND`s on it. You can drop it from any consumers.